### DISCUSSION TITLE
Xbox上无法查看收藏视频和直播，重启后无法登录，推荐页也空白

### DISCUSSION THREAD

**Discussion by EricAoki** [Richasy/Bili.Uwp#1624] [展示]
Xbox上无法查看收藏视频和直播，重启后无法登录，推荐页也空白

**Comment by EricAoki** [comment]
![image](https://user-images.githubusercontent.com/69636009/209932128-92b489d1-9878-4f5d-9507-58462c4caf9d.jpeg)

**Reply by gaoglgl** [reply]
我是win11，我也这样，不知道怎么回事

**Reply by EricAoki** [reply]
这项目估计是被b站刀了

**Reply by JiayanJohnnyChu** [reply]
用逍遥橙子版的客户端试了下，应该只有二维码登录才有问题，用户名密码方式登陆的还可以正常使用。但是rechasy的客户端用户名密码方式没法正常登录，验证码的界面被挡住了，没法使用。

**Reply by dustxn** [reply]
我也是这样

**Comment by Xpya233** [comment]
云之幻客户端的api都不能用了 他本人也弃坑了 还是用web版吧


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Richasy/Bili.Uwp"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年1月6日(星期五) 下午5:33
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [Richasy/Bili.Uwp] Xbox上无法查看收藏视频和直播，重启后无法登录，推荐页也空白 (Discussion #1624)





 
用逍遥橙子版的客户端试了下，应该只有二维码登录才有问题，用户名密码方式登陆的还可以正常使用。但是rechasy的客户端用户名密码方式没法正常登录，验证码的界面被挡住了，没法使用。
